CHANGELOG.rst

```diff
@@ -1,5 +1,17 @@
 Change log
 ==========
+1.4.0
```

Comment:
Add breaking keyword

Reply:
Added `[Breaking] due to the re-implementation of the ACL endpoint and the drop of the support of OSX and consul 1.1.0.` below as this line should just contain the version number (and an opt date)